pcsx2/gui/AppConfig.cpp

```diff
@@ -674,8 +674,9 @@ void AppConfig::LoadSave( IniInterface& ini )
 	GSWindow		.LoadSave( ini );
 	Framerate		.LoadSave( ini );
 #ifndef DISABLE_RECORDING
-	inputRecording.loadSave(ini);
+	inputRecording	.loadSave( ini );
```

Comment:
Formatting on this line was correct, it would be better to run clang on the entire file tbh.

Reply:
I'll just revert that change for now. We can do clang later.